src/components/MediaPicker.js

```diff
@@ -93,24 +99,25 @@ export const BCAnimationRenderer = ({
                 className="bc-remove-btn"
                 onClick={onRemoveClick}
             >
-                Remove File
+                {removeButtonLabel}
             </Button>
         </>
     ) : (
         <Button variant="secondary" onClick={onSelectClick}>
-            Select File
+            {selectButtonLabel}
         </Button>
     )
 };
 
-// TODO: find better name
-export const BCMediaPicker = ({
+export const BCMediaSelector = ({
```

Comment:
@deludd rename to just "MediaControl" and let's also remove the BC prefix for BCImage/Video/Animation/Renderer

Reply:
@deludd update the export in the ./components/index.js
https://tppr.me/Aki7mI

Reply:
@deludd also don't forget to rename the file